main.go

```diff
@@ -39,6 +40,10 @@ func init() {
 	}
 	restartThreshold = time.Duration(idleMinutes) * time.Minute
 	logDebug("idletime = " + restartThreshold.String())
+
+	ipAddress := GetOutboundIP().String()
```

Comment:
You mised this one after making the method private:
```suggestion
	ipAddress := getOutboundIP().String()
```

Reply:
Oh right! Sorry, I missed that out.